lib/cmd/fh2/account.js

```diff
@@ -265,28 +266,24 @@ function getFileType(file) {
 }
 
 function getDestination(file) {
-  var parts = file.split("/");
-  var dest = "";
-  for (var i = 0; i < parts.length; i++) {
-    var folder = parts[i].toLowerCase();
+  var folders = file.split(path.sep);
+  for (var i = 0; i < folders.length; i++) {
```

Comment:
I kept the regular `for` because of the short circuiting with `return`, there's no nice mechanism to do that with `forEach`. Other alternatives are also more complicated.